src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -127,10 +129,11 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
         <ModularTable
           columns={columnData}
           // Table will display at most (limit) entries.
-          data={tableData.length <= limit ? tableData : tableData.slice(0, -1)}
+          data={hasNextPage ? tableData.slice(0, -1) : tableData}
           emptyMsg={emptyMsg}
         />
       )}
+      {hasNextPage && auditLogsLoaded ? <AuditLogsTablePagination /> : null}
```

Comment:
I added the pagination to the bottom of the page as well as the top as it felt a bit unnatural to have to scroll back to the top of the page to navigation to the next page.

Reply:
That's a really good addition from a user experience perspective! However, while this is highly subjective, I was wondering if maybe making `AuditLogsTablePagination` sit on the bottom right would be "better" from a consistency standpoint? This way, both instances of this component would appear on the right hand side.

Reply:
I think this is a good suggestion, but as far as I'm aware all the pagination at the bottom of tables in our other products is either on the left or centred, so I might leave this where it is for consistency with our other products for now and maybe we can look at this again when this feature gets some input from design.